web/forms.py

```
@@ -100,6 +101,12 @@
 ]
 
 
+class SurveyForm(forms.ModelForm):
+    class Meta:
+        model = Survey
+        fields = ["title"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)